### DISCUSSION TITLE
Unable to select storage pool

### DISCUSSION THREAD

**Discussion by Yonny24** [tteck/Proxmox#266] [Q&A]
Hi,
Just got to say these scripts are awesome! Thank you!
Now I'm trying a plex container but having a daft issue trying to select the storage pool. No matter what I type on the keyboard nothing will select the pool I want. tried 'x' 'y' every character but nothing appears on the parenthesis.
How do I select the pool to continue on OK? (maybe a very simple answer)

![image](https://user-images.githubusercontent.com/64428686/172238548-fda96db2-c5a7-4bd5-a65b-92acc2b45494.png)

**Comment by tteck** [comment]
tab = move
space bar = select
